ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>@title - @start_time</title>
+  <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+
+  <script type="text/javascript" src="js/zepto.min.js"></script>
+  <script type="text/javascript">
+    window.META_COLUMNS = 8;
+  </script>
+  <script type="text/javascript" src="js/setup.js?@(resource_hashes['setup.js'])"></script>
+  <script type="text/javascript">
+    var QUERY_TRANSFORMS = {
+      'id': 'id=".*"',
+      'label': 'label=".*"',
+    };
+
+    function encoded_query_a_tag(query, title, content) {
+      document.write("<a href=\"?q=" + encodeURIComponent(query) + "\" title=\"" + title + "\">" + content + "</a>")
```

Comment:
By using single quotes for the outside strings you avoid requiring escaping.

Same below.